src/components/BillDisplay.js

```diff
@@ -0,0 +1,33 @@
+const BillDisplay = (props) => {
+    return (
+        <div className="bill-display-container">
+            <div className="gross-bill-per-person">
+                <div className="display-label-container">
+                        <p>Gross Bill</p>
```

Comment:
Take a look ate the indentation for line 6 and 7.

Reply:
Done